Core/src/org/sleuthkit/autopsy/discovery/search/DomainSearchCacheLoader.java

```diff
@@ -57,36 +57,38 @@
 
     @Override
     public Map<GroupKey, List<Result>> load(SearchKey key) throws DiscoveryException, SQLException, TskCoreException, InterruptedException {
-
         List<Result> domainResults = getResultDomainsFromDatabase(key);
-
         // Apply secondary in memory filters
         for (AbstractFilter filter : key.getFilters()) {
```

Comment:
This code is now duplicated.

Reply:
it appears this happened at the same time as the deleted conditional, and all revolves around me botching that merge conflict resolution. 

Will fix, thank you for catching, apologies for the headache.